backend/pkg/dbstore/report_content.go

```diff
@@ -15,59 +16,113 @@ type ReportContent struct {
 	StoreID     string `json:"storeID"`
 }
 
-func (datasource *SQLiteDatasource) GetReportContent(scheduleID string) ([]ReportContent, error) {
-	db, _ := sql.Open("sqlite3", datasource.Path)
-	defer db.Close()
+func ReportContentFields() string {
+	return "\n{\n\tID string\n\tScheduleID string\n\tPanelID string\n\tDashboardID string\n\tLookback int\n\tStoreID string\n}"
+}
```

Comment:
Not sure if it's worth it, but could this be defined as JSON and stringified? A little performance overhead, but a bit more elegant?

Reply:
Yeah I would like a nicer way, and not even sure if this is needed at all